nixos/modules/services/web-servers/keter/default.nix

```diff
@@ -0,0 +1,163 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.services.keter;
+in
+{
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ jappie ];
+  };
+  options.services.keter = {
+    enable = lib.mkEnableOption "Keter";
+    keterRoot = lib.mkOption {
+      type = lib.types.str;
+      default = "/var/lib/keter";
+      description = ''Mutable state folder for keter. In here the bundles get extracted,
+                      logs get written and the app's work folder is set to'';
+    };
+    keterPackage = lib.mkOption {
+      type = lib.types.package;
+      default = null;
```

Comment:
This is a `lib.types.package` but you are passing `null`. Explain?

Reply:
It needs to be set by the user, I couldn't figure out how to set this by default to `haskellPackges.keter`

Reply:
What does it do when you set it to `pkgs.haskellPackges.keter`? (And, if it works, also set defaultText ([example](https://github.com/nixos/nixpkgs/blob/3939c4bb8caf6458d1f4b2f1bacc221503d12afa/nixos/modules/config/users-groups.nix#L174)))